src/V3AstNodes.h

```diff
@@ -8956,7 +9002,10 @@ class AstCFunc final : public AstNode {
     AstScope* scopep() const { return m_scopep; }
     void scopep(AstScope* nodep) { m_scopep = nodep; }
     string rtnTypeVoid() const { return ((m_rtnType == "") ? "void" : m_rtnType); }
-    bool dontCombine() const { return m_dontCombine || isTrace() || entryPoint(); }
+    void rtnType(const string& rtnType) { m_rtnType = rtnType; }
+    bool dontCombine() const {
+        return m_dontCombine || isTrace() || entryPoint() || isConstructor();
```

Comment:
It was supposed to be `isConstructor()`, as `V3Combine` would remove the `Join` constructor because it was empty. But now I see I should put `_ctor_var_reset` in the constructor anyway, making this irrelevant.